### ISSUE TITLE
LuxMenuBar dropdowns should be keyboard navigable

### ISSUE THREAD

**Issue by maxkadel** [pulibrary/lux-design-system#445]
### Steps to recreate
1. Go to [LuxMenuBar in the Lux Styleguide](https://pulibrary.github.io/lux-design-system/#!/Components/LuxMenuBar)
2. Tab to one of the dropdowns
3. Open the dropdown using space bar
4. Attempt to navigate to one of the items in the dropdown using arrow keys (if there is a scroll-bar on the page it will scroll the page, otherwise it will appear to do nothing)

### WCAG Success Criterion
[2.1.1 - Keyboard navigable](https://www.w3.org/TR/WCAG22/#keyboard)

### WCAG Conformance level (A, AA, AAA)
A

### User impact
Difficult to navigate using keyboard

### Recommended fix
Arrow keys should navigate between menu items. Is it possible this is a focused-element issue?

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...
